atom/browser/api/atom_api_app.cc

```diff
@@ -378,9 +378,21 @@ void OnClientCertificateSelected(
     v8::Isolate* isolate,
     std::shared_ptr<content::ClientCertificateDelegate> delegate,
     mate::Arguments* args) {
+  if (args->Length() == 2) {
```

Comment:
I'm not sure I understand this check for `2` here.

Calling `callback(cert, null)` would hit here and return early and not use the cert  but calling `callback(cert, null, null)` would continue with the cert. 

What case is this targeting?